src/Fabulous.XamarinForms/Views/GestureRecognizers/DropGestureRecognizer.fs

```diff
@@ -0,0 +1,68 @@
+namespace Fabulous.XamarinForms
+
+open System.Runtime.CompilerServices
+open Fabulous
+open Xamarin.Forms
+
+type IDropGestureRecognizer =
+    inherit Fabulous.XamarinForms.IGestureRecognizer
+
+module DropGestureRecognizer =
+    let WidgetKey =
+        Widgets.register<DropGestureRecognizer> ()
+
+    let AllowDrop =
+        Attributes.define<bool>
+            "DropGestureRecognizer_AllowDrop"
+            (fun newValueOpt node ->
+                let gesture = node.Target :?> DropGestureRecognizer
+
+                let value =
+                    match newValueOpt with
+                    | ValueNone -> false
+                    | ValueSome v -> v
+
+                gesture.AllowDrop <- value)
+
+    let Drop =
+        Attributes.defineEvent<DropEventArgs>
+            "DropGestureRecognizer_Drop"
+            (fun target -> (target :?> DropGestureRecognizer).Drop)
+
+    let DragOver =
+        Attributes.defineEvent<DragEventArgs>
+            "DropGestureRecognizer_DragOver"
+            (fun target -> (target :?> DropGestureRecognizer).DragOver)
+
+
+    let DragLeave =
+        Attributes.defineEvent<DragEventArgs>
+            "DropGestureRecognizer_DragLeave"
+            (fun target -> (target :?> DropGestureRecognizer).DragLeave)
+
+
+[<AutoOpen>]
+module DropGestureRecognizerBuilders =
+    type Fabulous.XamarinForms.View with
+        static member inline DropGestureRecognizer<'msg>(onDrop: bool -> 'msg) =
+            WidgetBuilder<'msg, IDragGestureRecognizer>(
+                DropGestureRecognizer.WidgetKey,
+                DropGestureRecognizer.Drop.WithValue(fun args -> onDrop args.Handled |> box)
```

Comment:
I think we should return the `args` directly here.
It has both `Data` and `Handled` in it and both are useful.

https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.dropeventargs?view=xamarin-forms